testing/ssh-honeypot/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="ssh-honeypot"
+pkgver="0_git"
+_commit=b7baa9a6963e723ee0779a6cef0447739870ace3
+pkgrel=0
+pkgdesc="Fake sshd that logs ip addresses, usernames, and passwords."
+url="https://github.com/droberson/ssh-honeypot"
+arch="all"
+license="MIT"
+depends="openssh-keygen"
+makedepends="json-c0.12-dev libssh-dev"
+options="!check"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.zip::https://github.com/droberson/ssh-honeypot/archive/$_commit.zip
+	$pkgname.initd
+	$pkgname.confd"
+builddir="$srcdir/$pkgname-$_commit"
+
+
+package() {
+	mkdir -p "$pkgdir"/usr/bin/ "$pkgdir"/usr/share/ssh-honeypot
+	gcc -Wall -static-libgcc -o "$builddir"/bin/$pkgname "$builddir"/src/ssh-honeypot.c -lssh -ljson-c
```

Comment:
Do this in build please and use cc instead gcc. Also, can't the makefile do this?

Reply:
Yep, moved back to the Makefile.